posthog/settings/sentry.py

```diff
@@ -26,20 +26,28 @@ def traces_sampler(sampling_context: dict) -> float:
 
     if op == "http.server":
         path = sampling_context.get("wsgi_environ", {}).get("PATH_INFO")
+        referer = sampling_context.get("wsgi_environ", {}).get("HTTP_REFERER", None)
+
+        if referer.startswith("https://playground.posthog.net/"):
+            return 0
 
         # Ingestion endpoints (high volume)
-        if path.startswith(("/batch")):
-            return 0.00000001  # 0.000001%
+        if path.startswith("/batch"):
+            return 0.00000001  # 0.0000001%
         # Ingestion endpoints (high volume)
         elif path.startswith(("/capture", "/decide", "/track", "/s", "/e")):
-            return 0.0000001  # 0.00001%
+            return 0.0000001  # 0.000001%
         # Probes/monitoring endpoints
         elif path.startswith(("/_health", "/_readyz", "/_livez")):
             return 0.0001  # 0.01%
         # API endpoints
-        elif path.startswith(("/api/feature_flag")):
-            return 0.0001  # 0.01%
-        elif path.startswith(("/api")):
+        elif path.startswith("/api/projects") and path.endswith("/persons/"):
+            return 0.0001  # 0.001%
```

Comment:
You got to multiply by a 100 to get the percentage 😅 

```suggestion
            return 0.0001  # 0.01%
```

and same below

Reply:
Obviously I didn't copy other values without thinking about them :side-eye-monkey.gif: